### PR TITLE
chore: track crash fail rate

### PR DESCRIPTION
**Pull request by vladanpaunovic**
Adding Sentry SDK so we can track crash fail rate overall our releases

### Preview

Screenshot 2022-02-04 at 10 26 30

Screenshot 2022-02-04 at 10 27 20

### CODE REVIEW

File: src/sentry.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+const Sentry = require("@sentry/node");
+
+function initSentry({ inputs }) {
+  const config = {
+    dsn: "TBD",
```
Comment (vladanpaunovic) [inline src/sentry.js:5]: What's the best place to store DSN? Should it be a secret or not?

Comment (HazAT) [reply src/sentry.js:5]: You can use it here, np
It's not a secret

**Review by HazAT — CHANGES_REQUESTED** [review]

**Review by HazAT — APPROVED** [review]

**Comment by vladanpaunovic** [comment]
Update: we can test it with [release-tester](https://github.com/getsentry/release-tester/actions/workflows/release.yml) but only once merged

**Review by mitsuhiko — APPROVED** [review]

**Comment by BYK** [comment]
This is great. That said you know what we need as "the public"? SENTRY FOR GITHUB ACTIONS. PLEASE 🙏🏻

**Comment by BYK** [comment]
Eh, also why are you not putting `SENTRY_DSN` as a repo secret instead of passing it as custom args? :)

**Comment by vladanpaunovic** [comment]
> Eh, also why are you not putting `SENTRY_DSN` as a repo secret instead of passing it as custom args? :)

Oh yes! Great idea! Thanks a lot! :) 

> Eh, also why are you not putting `SENTRY_DSN` as a repo secret instead of passing it as custom args? :)

We are just trying it out atm but that's the idea.

P.S. We miss you @BYK 🥇

**Pull request merged**